layouts/shortcodes/EOL_manage.md

```diff
@@ -1,9 +1,11 @@
 <div class="admonition-warning"><p class="admonition-warning-title">Warning</p><div class="admonition-warning-text">
 
-Chef Manage is
-[deprecated](/versions.html#deprecated-products-and-versions) and does
-not work with Chef Automate 2.0. See our
-[documentation](https://automate.chef.io/docs/quickstart/) to learn more
-about Chef Automate 2. This document is no longer maintained.
+We currently support Chef Manage, however it is [deprecated](/versions/#deprecated-products-and-versions)
```

Comment:
Just a thought: "Chef Manage is deprecated an no longer under active development. It is supported on Chef Automate installations up to version 1.8 and replaced by Chef Automate versions [whatever version got matching features]. etc.,"